faq/_posts/2017-01-20-frats-and-sororities.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: How do fraternities and sororities work?
+author:
+  display_name: rmichnovicz
+  first_name: Randal
+  last_name: Michnovicz
+
+tags:
+    - Greek Life
+---
+
+I was clueless as to what to expect out of Georgia Tech Greek life, so after a
+bit more than a semester here, these are my observations.
+
+Check out Tech's official
+[Guide To Greek Life](http://greek.gatech.edu/uploads/files/2016GuidetoGreekLife%281%29.pdf)
+for info about social fraternities and sororities. Also see their official
+[grade and membership reports](http://greek.gatech.edu/content/4/grade-membership-reports).
+[Wikipedia](https://en.wikipedia.org/wiki/List_of_fraternities_and_sororities_at_Georgia_Institute_of_Technology)
+also maintains a list of social fraternities and sororities.
+
+There's a lot of selective organizations with Greek letters at most colleges,
+and Georgia Tech is no exception! There's 4 types of organizations I am
+referring to here: social fraternities/sororities, social
+fraternities/sororities, professional fraternities, and honors societies.
+The social ones that are managed by the Office of Greek Affairs, and the rest
+are basically clubs. I'll go over non-social Greek lettered orgs first, then
+talk about the social ones. While the word fraternity can mean either co-ed or
+male only, sororities are exclusive to women.
+
+Professional fraternties and organizations (all co-ed) are typically for forming
+connections and often skills. There are all sorts of organizations, from just
+general professional interests, to ones for industry fields, to
+ethnic/racial-interest clubs.
+
+Service organizations exist at all schools, but service fraternities are
+notable for being tight-knit groups bonded together over service. Alpha Phi
+Omega is a co-ed fraternity, and Omega Phi Alpha is a service sorority.
+
+Social fraternities and sororities exist to create deep and long-lasting
+friendships. These organizations are a big deal at Georgia Tech: 25% of
+undergraduates are a member of one! Every one of these groups is a member of one
+umbrella council: Interfraternity Council (IFC) (32 fraternities), Collegiate
+Panhellenic Council (CPC) (8 sororities + 2 associate chapters with separate
+recruitment from the rest), Multicultural Greek Council (4 cultural interest
+fraternities + 3 sororities), and National Pan-Hellenic Council (NPHC)
+(4 historically black fraternities + 4 sororities).
+
+Interfraternity Council fraternities are what people think of when one mentions
+frats. Despite their often large houses, Tech's frats are typically small to
```

Comment:
Overall it looks pretty complete, though. Can you replace "frats" with "fratenities" and such. I'll accept the PR then.